invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -98,7 +98,8 @@ <h2>{{ add_description.type.title_l10n }} <span
 {% macro _show_funding_item(item, index) %}
   {%- if item.award -%}
```

Comment:
You can move `<dt class="ui tiny header">{{ item.funder.name if item.funder }}</dt>` here if you want to switch the order and have the funder show up first.